pkg/config/config.go

```diff
@@ -103,6 +103,12 @@ func init() {
 	Datadog.SetDefault("auth_token_file_path", "")
 	Datadog.SetDefault("bind_host", "localhost")
 
+	// secrets backend
+	Datadog.BindEnv("secret_backend_command")
```

Comment:
Yes we dropped multiple backend: it would have add quite a lot of complexity the code and it's extremely simple to implement on the customer side if they need too.